src/app/editor.js

```diff
@@ -147,16 +151,15 @@ function Editor (loadingFromGist) {
 
   function setupStuff (files) {
     var untitledCount = '';
-    if (!files.length || window.localStorage['sol-cache']) {
+    if (files.length === 0) {
       if (loadingFromGist) return;
       // Backwards-compatibility
```

Comment:
Right, so I'm getting rid of all of the `sol-cache` stuff, which I assume is from an early version of browser-solidity. Is there more that we can remove?